### PR TITLE
Add Omise Link API

### PR DESCRIPTION
**Pull request by turboza**
#### 1. Objective reason

Add Link API, our new API, to Omise-PHP.

#### 2. Description of change

- Add `OmiseLink` with methods `create`, `retrieve` 

**Usage**

- Retrieve
```php
$links = OmiseLink::retrieve()
$link = OmiseLink::retrieve('link_id')
```

- Create
```php
$link = OmiseLink::create(array('amount'      => 100000,
                                'title'       => 'Order-384',
                                'description' => 'New Product'));
```

#### 3. Users affected by the change

None.

#### 4. Impact of the change

New Method.

#### 5. Priority of change

Normal.

#### 6. Alternate solution (if any)

None.

### CODE REVIEW

File: tests/omise/TestConfig.php [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <?php
 
 // Omise keys.
-define('OMISE_PUBLIC_KEY', 'pkey');
```
Comment (guzzilar) [inline tests/omise/TestConfig.php]: @turboza could you leave it as `pkey`?

File: tests/omise/TestConfig.php [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
-define('OMISE_SECRET_KEY', 'skey');
```
Comment (guzzilar) [inline tests/omise/TestConfig.php]: @turboza could you leave it as `skey`

File: tests/omise/TestConfig.php (hunk 1 above)
Comment (turboza) [inline tests/omise/TestConfig.php]: Fixed! 🔧

File: tests/omise/TestConfig.php (hunk 2 above)
Comment (turboza) [inline tests/omise/TestConfig.php]: Fixed! 🔧

**Comment by guzzilar** [comment]
LGTM 👍

**Comment by guzzilar** [comment]
btw, @turboza could you help me rebase `master` to this PR?
Thanks.

**Comment by turboza** [comment]
Rebased, merging soon if no more comment.

**Pull request merged**